### DISCUSSION TITLE
Box Shadow - a sneak peek

### DISCUSSION THREAD

**Discussion by bph** [WordPress/developer-blog-content#224] [Topic Ideas]
Box shadow were available to theme developers since 6.2 

WordPress 6.5 will bring a considerable update including updated UI for content creators. 

This article will explain the new features and how to use them. It also covers what will be next for box shadow support beyond. 

A draft is available in [this Google Doc. ](https://docs.google.com/document/d/1kmpktk4EkSRSxBCK70C1lYWOqjAcvd1N4qvOhKvMthM/edit)

The article will also serve as an update to the previous post on Box Shadow support: [Using the box shadow feature for themes](https://developer.wordpress.org/news/2023/01/12/using-the-box-shadow-feature-for-themes/)

**Comment by justintadlock** [comment]
This topic is approved and a ticket created here: https://github.com/WordPress/developer-blog-content/issues/236

**Comment by bph** [comment]
Published: [Drop Shadow updates in WordPress 6.5](https://developer.wordpress.org/news/2024/03/21/drop-shadow-updates-in-wordpress-6-5/)
